app/controllers/hyrax/dashboard/collections_controller.rb

```diff
@@ -220,6 +220,7 @@ def valkyrie_create
           @collection = transactions['change_set.create_collection']
                         .with_step_args(
                           'change_set.set_user_as_depositor' => { user: current_user },
+                          'change_set.add_to_collections' => { collection_ids: Array(params[:parent_id]) },
```

Comment:
Fixes failure to set parent collection in test: `#create when params includes parent_id`